docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -923,6 +923,65 @@ public class CustomLoadBalancerConfiguration {
 	}
 ----
 
+=== Instance Health-Check for LoadBalancer
+
+It is possible to enable a scheduled HealthCheck for the LoadBalancer. The `HealthCheckServiceInstanceListSupplier`
+is provided for that. It regularly verifies if the instances provided by a delegate
+`ServiceInstanceListSupplier` are still alive and only returns the healthy instances,
+unless there are none - then it returns all the retrieved instances.
+
+TIP: This mechanism is particularly helpful while using the `SimpleDiscoveryClient`. For the
```

Comment:
I think we also need to add some documentation about when the `SimpleDiscoveryClient` is used.  I dont see that in the current docs.

Reply:
It's a good idea, but I would add it in the `DiscoveryClient` section, not necessarily as a part of this PR.